.env

```diff
@@ -1,20 +1,20 @@
 NODE_ENV='production'
-ACCESS_TOKEN_COOKIE_NAME=null
-BASE_URL=null
-CREDENTIALS_BASE_URL=null
-CSRF_TOKEN_API_PATH=null
-ECOMMERCE_BASE_URL=null
-LANGUAGE_PREFERENCE_COOKIE_NAME=null
-LMS_BASE_URL=null
-LOGIN_URL=null
-LOGOUT_URL=null
-LOGO_URL=null
-LOGO_TRADEMARK_URL=null
-LOGO_WHITE_URL=null
-FAVICON_URL=null
-MARKETING_SITE_BASE_URL=null
-ORDER_HISTORY_URL=null
-REFRESH_ACCESS_TOKEN_ENDPOINT=null
-SEGMENT_KEY=null
-SITE_NAME=null
-USER_INFO_COOKIE_NAME=null
+ACCESS_TOKEN_COOKIE_NAME=
```

Comment:
does this then get processed as an empty string? I'm just so used to this type of empty definition just breaking things. I cannot find anything about how to properly handle nulls in dotenv files though (just a lot of people complaining about it).

Ie is there a reason not to just put actual empty strings here?

Reply:
Agree this is kinda freaky-lookin' :)

Reply:
Oof, yeah, I think I'll change it.  I just saw some stuff about blank variables getting dropped instead of becoming empty string, so I think I won't take any chances.

Reply:
(my expectation was that blank is translated to an empty string, but if it's confusing/concerning anyway, might as well make it clearer)

Reply:
I guess based on your prior comment, is our goal for the values to not be picked up at all rather than the app config code interpreting their empty string value as falsy? I think I was mostly concerned about this actual thing that happened: https://github.com/bkeepers/dotenv/issues/339#issuecomment-383588289 (but that's Ruby)
Supposedly the JS dotenv does support blank definitions:
```
empty values become empty strings (EMPTY= becomes {EMPTY: ''})
```